packages/@glimmer/util/lib/weak-set.ts

```diff
@@ -0,0 +1,18 @@
+export default (typeof WeakSet === 'function'
```

Comment:
This is moving upstream from Ember, we'll be sure to remove it there when we integrate these changes